doc/btp_spec.txt

```diff
@@ -365,9 +365,10 @@ Commands and responses:
 					Use own ID address (1 octet)
 		Return Parameters:	<none>
 
-		Valid Address_Type parameter values:
-					0x00 = Public
-					0x01 = Random
+		Valid Own_Addr_Type parameter values:
```

Comment:
Address_Type is a different parameter, so just add a note about Own_Addr_Type below this.

Reply:
Thanks, I've missed that line.